sonic-net/sonic-buildimage
# update submodule sonic-utilities

#### Why I did it
update submodule sonic-utilities to support multi-asic for show queue counters

#### How I did it
update submodule sonic-utilities

#### How to verify it
Verify it by using show CLI and also queuestat script, done in sonic-utilities PR https://github.com/sonic-net/sonic-utilities/pull/2439

#### Which release branch to backport (provide reason below if selected)

 required for 202205 since it is the branch for T2.


#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
